src/components/common/Dialog/store/dialogSlice.js

```diff
@@ -0,0 +1,45 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const FormType = Object.freeze({
+  rejestracja: 'rejestracja',
+  zalozonyProfil: 'zalozonyProfil',
+  loginDialog: 'loginDialog',
+  rejestracjaEmail: 'rejestracjaEmail'
+
+})
+
+const initialState = {
+  isOpen: false,
+  formType: undefined,
+};
+
+const initialLogin = {
+  login: false
+}
+
+const slice = createSlice({
+  name: "dialog",
+  initialState,
+  reducers: {
+    openDialog: (state, action) => {
+      return { ...action.payload, isOpen: true }
+    },
+    closeDialog: (state) => {
+      state.isOpen = false;
+      state.formType = undefined;
+    },
+    login: (state, action) => {
+      state.login = true;
+      state.name = action.payload.name;
+    },
+    logout: (state, action) => {
+      state.login = false;
+      state.name = undefined;
+    }
```

Comment:
To można usunąć, nie jest nigdzie używane.

Reply:
usunięte